src/ctapipe/io/datawriter.py

```diff
@@ -353,19 +475,21 @@ def _write_constant_pointing(self, event):
         So we write the first pointing information for each telescope into the
         configuration table.
         """
-        obs_id = event.index.obs_id
-
-        for tel_id, pointing in event.pointing.tel.items():
-            if tel_id in self._constant_telescope_pointing_written[obs_id]:
-                continue
-            index = TelescopeConfigurationIndexContainer(
-                obs_id=obs_id,
-                tel_id=tel_id,
-            )
-            self._writer.write(
-                f"configuration/telescope/pointing/tel_{tel_id:03d}", (index, pointing)
-            )
-            self._constant_telescope_pointing_written[obs_id].add(tel_id)
+        obs_id = tel_event.index.obs_id
+        tel_id = tel_event.index.tel_id
+
+        if tel_id in self._constant_telescope_pointing_written[obs_id]:
+            return
+
+        index = TelescopeConfigurationIndexContainer(
+            obs_id=obs_id,
+            tel_id=tel_id,
+        )
+        self._writer.write(
+            f"configuration/telescope/pointing/tel_{tel_id:03d}",
```

Comment:
shouldn't this use `self.table_name` to construct a table name?

Reply:
Good catch. Fixed.